app/controllers/people_controller.rb

```diff
@@ -5,20 +5,22 @@ class PeopleController < ApplicationController
 
   def signin
     #This is the sign-in sheet, not anything about authentication
-    @people = Person.active
-    @department = Department.where(name: "Police")&.first
+    @department = Department.where(shortname: "BAUX").first
```

Comment:
It would actually not be a good time, because this PR should not have any refactoring in it. This PR does not represent what I want development to be, but instead represents what Production *is*. I needed to make these changes, or the app would not function.
However, I am *very* anxious to remove these hard-coded references, and support any effort towards that. Just not in _this_ PR. I want to have a working point I can tag.